### ISSUE TITLE
Avatars in examples are missing aria-label

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#26208]
Make sure Table and DataGrid examples that use avatars have aria-labels on the avatar component so that screen reader users know what the images are

https://react.fluentui.dev/?path=/docs/preview-components-table--default#cell-actions
https://react.fluentui.dev/?path=/docs/preview-components-datagrid--default
